spec/models/room_spec.rb

```diff
@@ -1,7 +1,31 @@
 require "rails_helper"
 
 RSpec.describe Room, type: :model do
+  before :each do 
+    @hotel = Hotel.create!(name: "Echo Mountain Inn", location: "Echo Mountain")
+    @guest_1 = Guest.create!(name: "Charlize", nights: 3)
+    @room_1 = @hotel.rooms.create!(rate: 100, suite: "Presidential Suite")
+    @guest_room_1 = GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_1.id)
+  end
   describe "relationships" do
     it { should belong_to :hotel }
   end
+
+  describe "relationships" do 
+    it { should have_many :guest_rooms }
+  end
+
+  describe "relationships" do
+    it { should have_many(:guests).through(:guest_rooms) }
+  end
```

Comment:
It's fairly common to see all of these `it` blocks on lines 11, 15, and 19 together in one `describe 'relationships'` block. Feel free to do that if you'd like!